### ISSUE TITLE
Request: Add version info to usage output

### ISSUE THREAD

**Issue by mejofi** [sesh/ready#29]
**Version**: 1.2.4

Right now there does not seem to be a way to easily get the version you're currently running, at least not from within `ready`. I would expect it to be present in the usage output, for example, which currently does not have it, or perhaps a `--version` flag?
```
ready

Usage: ready.py [--headers] [--content] [--json] [--quiet] [--score] [--fuzz] <domain>

  --headers      Output the headers from the HTTPS request made to the domain
  --content      Output the content from the HTTPS request made to the domain
  --fuzz         Include checks that fuzz urls (only run this on your own domain)
  --json         Provide JSON output
  --quiet        No text output
  --score        Print a score out of 100 for this domain
  --doc          Print the list of check names

Development / experimental options for filtering checks and HTTP requests during testing:

  --check-filter=<x>     Only run checks that match the provided filter
  --request-filter=<x>   Only make HTTP requests that match the provided filter
```

**Comment by sesh** [comment]
Ooooh, I was slightly ahead of you on this one (finally!).

I added `--version` to the options over the weekend. I'll release a new version today that includes it. The version will also be added to the JSON output.

**Comment by sesh** [comment]
Should be available in `v1.2.5` once the action publishes it.

**Comment by mejofi** [comment]
```
$ ready --version 
ready 1.2.5
```

Nice! 🙂
